opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchQueryTest.java

```diff
@@ -32,7 +33,11 @@
 public class MatchQueryTest {
   private final DSL dsl = new ExpressionConfig().dsl(new ExpressionConfig().functionRepository());
   private final MatchQuery matchQuery = new MatchQuery();
-  private final FunctionName match = FunctionName.of("match");
+  private final FunctionName matchName = FunctionName.of("match");
+  private final FunctionName matchQueryName = FunctionName.of("matchquery");
+  private final FunctionName matchQueryWithUnderscoreName = FunctionName.of("match_query");
+  private final FunctionName[] functionNames =
+      {matchName,matchQueryName, matchQueryWithUnderscoreName};
```

Comment:
Can you add a test which builds `matchQuery` with different function names and checks that queries are same?

Reply:
Added as an IT test in `MatchIT.java` in https://github.com/Bit-Quill/opensearch-project-sql/pull/163/commits/77fcacd1f3501b6d9fcb9fad25f407edea0fa6ef